src/celeritas_cmake_strings.hh.in

```diff
@@ -3,7 +3,7 @@
  * See the top-level COPYRIGHT file for details.
  * SPDX-License-Identifier: (Apache-2.0 OR MIT)
  *---------------------------------------------------------------------------*/
-/*! \file celeritas_cmake_strings.h
+/*! \file celeritas_cmake_strings.hh
```

Comment:
Since these are C++ files now, you can change the comment markers, update the file type in the first line, and use `#pragma once`.